make build-docs fails in container missing d2

On a fresh `prairielearn/prairielearn` container, `make build-docs` fails unable to find the `d2` command.

```shell
bash-5.2# make build-docs
/bin/sh: line 1: d2: command not found
make: *** [Makefile:99: build-docs] Error 127
```

--- a/Makefile
+++ b/Makefile
@@ -100,4 +100,4 @@ build-docs:
 	@/tmp/pldocs/venv/bin/python3 -m pip install -r docs/requirements.txt
 	@/tmp/pldocs/venv/bin/python3 -m mkdocs build --strict
 
-ci: lint typecheck check-dependencies test build-docs
+ci: lint typecheck check-dependencies test

